code/modules/antagonists/blob/structures/node.dm

```diff
@@ -27,11 +31,19 @@
 	add_overlay(blob_overlay)
 	add_overlay(mutable_appearance('icons/mob/blob.dmi', "blob_node_overlay"))
 
-/obj/structure/blob/node/Destroy()
+/obj/structure/blob/special/node/creation_action()
+	if(overmind)
+		overmind.node_blobs += src
+
+/obj/structure/blob/special/node/Destroy()
 	GLOB.blob_nodes -= src
 	STOP_PROCESSING(SSobj, src)
+	if(overmind)
+		overmind.node_blobs -= src
```

Comment:
Is there a possibility of the `overmind` to be destroyed before this structure? If so, do these two variables (`overmind` and `node_blobs`) get properly cleaned?

Reply:
Yes, they do